build_runner/web/hot_reload_client.dart

```diff
@@ -12,9 +12,9 @@ import 'dart:html';
 import 'package:js/js.dart';
 import 'package:js/js_util.dart';
 
-import 'module.dart';
-import 'reload_handler.dart';
-import 'reloading_manager.dart';
+import 'package:build_runner/src/server/build_updates_client/module.dart';
```

Comment:
We actually might need to move these out to web?

Reply:
They are imported by some other stuff too I think. I didn't look _super_ closely so maybe they are just all in a little import cycle...

Reply:
I think the only blocker from moving these is some imports from tests.

We discussed offline and we'll ship this as is. In the next breaking release of `build_runner` we'll drop the hot reload code entirely in favor of the implementation in `webdev`